client/layout/guided-tours/tours/jetpack-monitoring-tour.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const JetpackMonitoringTour = makeTour(
+	<Tour name="jetpackMonitoring" version="20180611">
+		<Step
+			name="init"
+			target=".jetpack-monitor-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>
+						{ translate(
+							"Let's enable monitoring of your site's uptime " +
+								'by activating the toggle switch for Jetpack Monitor.'
+						) }
+					</p>
+					<ButtonRow>
+						<Continue
+							target=".jetpack-monitor-settings .form-toggle__switch"
+							step="finish"
+							click
+							hidden
+						/>
+						<SiteLink href="/checklist/:site">{ translate( 'Return to the checklist' ) }</SiteLink>
+					</ButtonRow>
+				</Fragment>
+			) }
+		</Step>
+
+		<Step name="finish" placement="right">
+			{ ( { translate } ) => (
+				<Fragment>
+					<h1 className="tours__title">
+						<span className="tours__completed-icon-wrapper">
+							<Gridicon icon="checkmark" className="tours__completed-icon" />
+						</span>
+						{ translate( 'Excellent, you’re done!' ) }
+					</h1>
+					<p>
+						{ translate(
+							'Uptime Monitoring has been enabled. Would you like to continue setting up the security essential features for your site?'
```

Comment:
Can we change "Uptime Monitoring" to "Jetpack Monitor" here??

Reply:
Sure! Also, I just realized that if anything, it should've been _Down_time Monitoring.

(FWIW, I tried to make this consistent with the UI controls, where there isn't really any mention of Jetpack.)

Reply:
Ah, that's a great point! I do recall from various past PRs that we were trying to move away from naming features and towards than more descriptive names. Let's use "Downtime Monitoring" across the board, since that's what we use on jetpack.com/pricing. 

I'll coordinate with Jeff to make sure we're consistent across our Marketing side of things too.